File: src/main/java/org/example/products/manufacturer/ManufacturerBuilder.java
package org.example.products.manufacturer;

public class ManufacturerBuilder {
    private String name;
    private String country;

    public ManufacturerBuilder() {
        this.name = "Default";
        this.country = "Default";
    }

    public ManufacturerBuilder setName(String name) {
        this.name = name;
        return this;
    }

    public ManufacturerBuilder setCountry(String country) {
        this.country = country;
        return this;
    }

    public Manufacturer build() {
        return new Manufacturer(name, country);
    }
}

File: src/main/java/org/example/products/manufacturer/ManufacturerBuilder.kt
package org.example.products.manufacturer

class ManufacturerBuilder {
    private var name: String = "Default"
    private var country: String = "Default"

    fun setName(name: String): ManufacturerBuilder {
        this.name = name
        return this
    }

    fun setCountry(country: String): ManufacturerBuilder {
        this.country = country
        return this
    }

    fun build(): Manufacturer {
        return Manufacturer(name, country)
    }
}